alternator/executor.cc

```diff
@@ -2811,7 +2819,7 @@ future<executor::request_return_type> executor::batch_write_item(client_state& c
         co_await verify_permission(_enforce_authorization, client_state, b.first, auth::permission::MODIFY);
     }
 
-    _stats.api_operations.batch_write_item_batch_total += batch_size;
+    _stats.api_operations.batch_write_item_batch_total += total_items;
```

Comment:
@amnonh please review that this change didn't break anything in your batch size calculation.
@radoslawcybulski needed to have the number of items earlier in the code, so after he already calculates there was no need for the "batch_size" calculation that you did.